test/functional/p2p_invalid_tx.py

```diff
@@ -93,21 +93,22 @@ def run_test(self):
         self.reconnect_p2p(num_connections=2)
 
         self.log.info('Test orphan transaction handling ... ')
+        SCRIPT_PUB_KEY_OP_TRUE = b'\x51\x75' * 15 + b'\x51'
```

Comment:
I specifically removed "magic bytes" from this test in https://github.com/PIVX-Project/PIVX/pull/2360/commits/48e5d54b5533404e29952f09e53275b2a4d2abde. Please, let's not re-introduce them. x51 is `OP_TRUE`, and x75 is `OP_DROP`.

Reply:
yeah k. I forgot the script constants existence in the functional test framework.